rllib/offline/dataset_reader.py

```diff
@@ -20,6 +77,9 @@ class DatasetReader(InputReader):
             "input_config"={
                 "format": "json",
                 "path": "/tmp/sample_batches/",
+                # By default, parallelism=num_workers.
+                "parallelism": 3,
+                "num_cpus_per_read_task": 0.5,
```

Comment:
Just a question for my understanding: CPUs=0.5 means this is not fully parallelized, then, just like python-threading (of having two processes run on same CPU)?

Reply:
oh, Dataset spins up a Ray task for each shard, reading all the data into memory.
this basically means 2 such shard readers can be scheduled on a single CPU.
0.5 is the default that Dataset uses.
I think for smaller RL dataset, we can even use 0.1, or 0.01 actually.